bitnami/wordpress/templates/deployment.yaml

```diff
@@ -84,9 +84,9 @@ spec:
             - |
               mkdir -p /bitnami/wordpress
               {{- if eq ( toString ( .Values.volumePermissions.containerSecurityContext.runAsUser )) "auto" }}
-              find /bitnami/wordpress -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs chown -R $(id -u):$(id -G | cut -d " " -f2)
+              chown -R $(id -u):$(id -G | cut -d " " -f2) /bitnami/wordpress
```

Comment:
We should definitely keep excluding some files that are common in mounted volumes. I mean, keeping this part: `-not -name ".snapshot" -not -name "lost+found`.